conversationstarters.py

```diff
@@ -0,0 +1,31 @@
+import json
+import logging
+import random
+from pathlib import Path
+
+from discord.ext import commands
+
+log = logging.getLogger(__name__)
+
+with open(Path('bot', 'resources', 'easter', 'starter.json'), 'r', encoding="utf8") as f:
+    starters = json.load(f)
+
+
+class ConvoStarters(commands.Cog):
+    """A cog which posts easter conversation starters"""
+
+    def __init__(self, bot):
+        self.bot = bot
+
+    @commands.command(aliases=('conversation_starters', 'convo_starters'))
+    async def convo_starter(self, ctx):
+        """Responds with a random conversation starter"""
+
+        await ctx.send(f"{random.choice(starters['starters'])}")
+
+
+def setup(bot):
+    """Conversation starters Cog load."""
+
+    bot.add_cog(ConvoStarters(bot))
+    log.info("Conversation starters cog loaded")
```

Comment:
The log should say the cog class name so it's easy to know exactly what object it's referring to and can be easily searched elsewhere in logs or the project when debugging if needed.
```suggestion
    log.info("ConvoStarters cog loaded")
```